pkg/local_workflows/whoami_workflow.go

```diff
@@ -56,7 +56,7 @@ func whoAmIWorkflowEntryPoint(invocationCtx workflow.InvocationContext, _ []work
 
 	// define userme API userMeEndpoint
 	baseUrl := config.GetString(configuration.API_URL)
-	url := baseUrl + apiVersion + userMeEndpoint
+	url := fmt.Sprintf("%s%s?version=%s", baseUrl, restSelfEndpoint, apiVersion)
 
 	// call userme API userMeEndpoint
 	userMe, err := fetchUserMe(httpClient, url, logger)
```

Comment:
**suggestion:** We discussed during a call, move fetchUserMe across to `api.go` so that is collocated with our other Snyk API transactions. 

Also look at bumping the endpoint version of /rest/self used by the CLI to a more recent version.